.github/workflows/katana.yml

```
@@ -134,10 +133,20 @@ jobs:
           verbose: true
 
   deploy-beta:
-    name: Beta deployment
-    runs-on: ubuntu-latest
+    name: Beta deployment (${{ matrix.name }})
+    runs-on: ${{ matrix.runner }}
     if: github.ref == 'refs/heads/develop'
     needs: [ static-analysis, unit-tests ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)